core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -75,6 +74,18 @@ class CampaignsResource(resolver: ExternalResolverClient, db: Database)
   }
 
   val route = path("install_campaigns") {
+
+    import org.genivi.sota.CirceSupport._
+    import org.genivi.sota.refined.SprayJsonRefined.{refinedUnmarshaller, refinedFromRequestUnmarshaller}
+    import io.circe._
+    import org.joda.time.DateTime
+    import org.joda.time.format.ISODateTimeFormat
+
+    implicit val dateTimeEncoder : Encoder[DateTime] = Encoder.instance[DateTime]( x =>  Json.string( ISODateTimeFormat.dateTime().print(x)) )
+    implicit val dateTimeDecoder : Decoder[DateTime] = ???
```

Comment:
It can't be removed (entity needs it), and there is no test to check if the implementation is correct -- so I think it's better to leave it and have it crash the test whenever somebody adds it?